08_deploy_bmo.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
+
+source bmo_install_prereqs.sh
+# Get the latest bits for baremetal-operator
+go get github.com/metalkube/baremetal-operator
+pushd ~/go/src/github.com/metalkube/baremetal-operator
+
+# Start deploying on the new cluster
+
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/service_account.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/role.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/role_binding.yaml
+oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
+
+export OPERATOR_NAME=baremetal-operator
+operator-sdk up local --namespace=bmo-project
```

Comment:
This is going to run the operator on the local host, and block the installer. You can replace lines 15 and 16 with something like

  oc --as system:admin --config ocp/auth/kubeconfig apply -f deploy/operator.yaml

That will launch the most recent version of the operator using the image referenced in the operator.yaml file.